packages/wonder-blocks-dropdown/src/components/__tests__/action-menu.test.tsx

```diff
@@ -51,14 +55,18 @@ describe("ActionMenu", () => {
         );
 
         // Act
-        userEvent.tab();
-        userEvent.keyboard("{enter}");
+        await userEvent.tab();
+        await userEvent.keyboard("{enter}");
 
         // Assert
-        expect(screen.getByRole("menu")).toBeInTheDocument();
+        expect(
+            await screen.findByRole("menu", {hidden: true}),
+        ).toBeInTheDocument();
     });
 
-    it("closes itself on escape", () => {
+    // TODO(FEI-5533): Key press events aren't working correctly with
+    // user-event v14. We need to investigate and fix this.
```

Comment:
**question:** What's happening in this case? is the focus order being set incorrectly?